sources/osgGA/FirstPersonManipulator.js

```diff
@@ -56,14 +57,17 @@ FirstPersonManipulator.prototype = MACROUTILS.objectInherit( Manipulator.prototy
         this._zoom = new OrbitManipulator.Interpolator( 1 );
 
         this._stepFactor = 1.0; // meaning radius*stepFactor to move
-        this._target = Vec3.create();
         this._angleVertical = 0.0;
         this._angleHorizontal = 0.0;
 
         // tmp value use for computation
         this._tmpGetTargetDir = Vec3.create();
 
-        this._rotBase = Matrix.create();
+        // vr controls
+        this._vrEnable = false;
```

Comment:
Agree we would probably needs something in the middle like simple internal representation but with helpers/config to set rotation/position in the manipulator.
I will merge it like that for now